GeometryCore/include/Polygon.h

```diff
@@ -44,6 +44,7 @@ namespace geom_utils
         }
 
         void add(const FPoint2D& p) { points.push_back(p); }
+        void addRange(const std::vector<FPoint2D>& p) { std::for_each(p.begin(), p.end(), [&](auto el) { points.push_back(el); }); }
```

Comment:
`std::copy(p.begin(), p.end(), std::back_inserter(points));` would be a bit neater, isn't it?)
`copy` is exactly what we want to do here

Reply:
yes, it is better. i will implement it